tests/test_spatial_methods/test_finite_volume.py

```diff
@@ -854,73 +854,112 @@ def test_indefinite_integral(self):
             "positive particle": pybamm.FiniteVolume,
         }
         disc = pybamm.Discretisation(mesh, spatial_methods)
-        # lengths
-        ln = mesh["negative electrode"].edges[-1]
 
-        # macroscale variable
-        var = pybamm.Variable("var", domain=["negative electrode", "separator"])
+        # input a phi, take grad, then integrate to recover phi approximation
```

Comment:
could you add a comment here that this is the only way to test because the integrand has to have a grad?